tools/New-AzureSampleResources.ps1

```diff
@@ -0,0 +1,141 @@
+param
+(
+    [Parameter(Mandatory = $true)]
+    [string] $AzureSubscriptionId,
+    [Parameter(Mandatory = $false)]
+    [switch] $NodeJS,
+    [Parameter(Mandatory = $false)]
+    [switch] $Golang,
+    # This variable is used to load existing service principal details from another powershell script. For the existing details, the .ps1 file needs to contain
+    # a hashtable for the service principal application and certificate hashtable with the following keys accessible from this script:
+    # $azureAppSpInfo["ClientId"]
+    # $azureAppSpInfo["ClientSecret"]
+    # $azureAppSpInfo["ObjectId"]
+    # $azureCertSpInfo["ClientId"]
+    # $azureCertSpInfo["CertificateThumbprint"]
+    # $azureCertSpInfo["ObjectId"]
+    [Parameter(Mandatory = $false)]
+    [string] $ExistingServicePrincipalDetails
+)
+
+. "C:\CloudDeployment\BVTs\BVTSettingUtils.ps1"
+Import-Module $([io.path]::combine($PSScriptRoot, "Common.psm1"))
```

Comment:
e.g. common.psm1 is not going to be found in github. you can keep it in your fork